migration/1643716017830-ProjectTotalReactions.ts

```diff
@@ -0,0 +1,25 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class ProjectTotalReactions1643716017830 implements MigrationInterface {
+  async up(queryRunner: QueryRunner): Promise<void> {
+    const [projectTableExists, projectUpdateTableExists, reactionTableExists] =
+      await Promise.all([
+        queryRunner.hasTable('project'),
+        queryRunner.hasTable('project_update'),
+        queryRunner.hasTable('reaction'),
+      ]);
+
+    if (projectTableExists && reactionTableExists)
+      await queryRunner.query(
```

Comment:
I tried running my migrations locally. But they fail. I think we should add an Alter table query, I have to run the server first.

In staging/prod migrations are run first, so they will break here as the tables do exist, so a similar scenario to my local will occur, breaking the deploy.

It doesn't occur on CI/CD because as Mohammad explained me, it runs on a brand new db. So the tables don't exist in that exact instance, hence doesn't break. 

I think we should add something like:
```
      `ALTER TABLE IF EXISTS  "project" ADD COLUMN IF NOT EXISTS "totalReactions" integer DEFAULT 0`
      `ALTER TABLE IF EXISTS  "project_update" ADD COLUMN IF NOT EXISTS "totalReactions" integer DEFAULT 0`,
```

@mohammadranjbarz  can confirm.